stdlib/SparseArrays/test/sparse.jl

```diff
@@ -519,6 +519,7 @@ dA = Array(sA)
         @test lmul!(Diagonal(bi), copy(dA)) ≈ ldiv!(Diagonal(b), copy(sA))
         @test lmul!(Diagonal(bi), copy(dA)) ≈ ldiv!(transpose(Diagonal(b)), copy(sA))
         @test lmul!(Diagonal(conj(bi)), copy(dA)) ≈ ldiv!(adjoint(Diagonal(b)), copy(sA))
+        @test Diagonal(b) \ copy(sA) == ldiv!(Diagonal(b), copy(sA))
```

Comment:
No need for the copy, and I'd suggest so check for sparsity.
```suggestion
        @test Diagonal(b) \ sA == ldiv!(Diagonal(b), copy(sA))
        @test issparse(Diagonal(b) \ sA)
```